app/models/ont/flowcell.rb

```diff
@@ -3,21 +3,26 @@
 module Ont
   # Ont::Flowcell
   class Flowcell < ApplicationRecord
-    # Currently redundant, will need to be re-implemented soon
     include Uuidable
 
+    # Run has multiple of these flowcells up to the maximum number for the instrument.
     belongs_to :run, foreign_key: :ont_run_id, inverse_of: :flowcells
-    belongs_to :library, foreign_key: :ont_library_id, inverse_of: :flowcell
-    delegate :requests, to: :library
 
+    # We have changed the model to use pool rather than library.
+    # We assume one-to-one relationship with pool at the moment.
+    # belongs_to :library, foreign_key: :ont_library_id, inverse_of: :flowcell
```

Comment:
Remove extra comments in this file that aren't needed.

Reply:
I have removed the extra comments.